common/changes/@microsoft/rush/benpapp-upgrade-ts-4.7_2022-08-02-20-58.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Upgrading TypeScript dependency to 4.7",
```

```suggestion
      "comment": "",
```

Consumers of this package won't care.